assembly/channel/index.ts

```diff
@@ -83,12 +187,22 @@ export const receive_length_pointer = memory.data(sizeof<u32>());
     return buffer;
   }
 
-  // send some data
+  /**
+   * Send a given T to the Channel to be received elsewhere.
```

Comment:
It would be worth adding here that if you use a bounded channel and it's full the sending will block the execution until there is space available in the channel.

Reply:
Wow. I didn't know that. Thanks for the information.